### ISSUE TITLE
[BUG] admiralty-type-ahead prediction box pushes elements down instead of overlaying like the admiralty-select

### ISSUE THREAD

**Issue by developernm** [UKHO/admiralty-design-system#53]
admiralty-type-ahead prediction box pushes elements down instead of overlaying like the admiralty-select

```
<admiralty-type-ahead type="text"
                                      placeholder="Commissioning organisation"
                                      [label]="'Commissioning organisation *'">
  <ng-container *ngFor="let organisation of commissioningOrganisations$ | async" >
    <admiralty-type-ahead-item [value]="organisation.organisationName"></admiralty-type-ahead-item>
  </ng-container>
</admiralty-type-ahead>
```
![image](https://github.com/UKHO/admiralty-design-system/assets/24655342/3ecfa055-bda6-4c48-a96c-11a4863432c4)

**Comment by developernm** [comment]
@liamfarley13 @LukeTaylor34

**Comment by ecstaticrainbow** [comment]
Fixed in #40
